### ISSUE TITLE
dib-lab GitHub account with code

### ISSUE THREAD

**Issue by johnsolk** [dib-lab/2018-paper-reanalysis-op-ed#6]
@ctb how do we modify the [Zenodo record](https://zenodo.org/record/249982#.Ww7uF1MvyL8) created through the dib-lab GitHub, where the code is? Did you create the zenodo record last year? 

I don't seem to have access to it on my Zenodo account like I do to my own GitHub repositories. It does say "If your organization's repositories do not show up in the list, please ensure you have [enabled third-party access to the Zenodo application](https://help.github.com/articles/approving-oauth-apps-for-your-organization/). Private repositories are not supported." But if I created the record last year, it should have already been enabled. Confused.

Would like to change my last name -> Johnson

Also, if we do modify the code, I would like to be able to have access to that.

**Comment by halexand** [comment]
I don't have access either... 

I think the name change might happen if you change your name in your profile? Have you tried that? https://zenodo.org/account/settings/profile/

**Comment by johnsolk** [comment]
Yes, that is how I changed the name on the Zenodo records that I created. I also changed it on GitHub.

**Comment by johnsolk** [comment]
@ctb figure it out. We released a v2 of the code (I updated some of the DOI on the README.md). This updated my last name in the zenodo record. We have a version 2 on that record now. 

[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1256189.svg)](https://doi.org/10.5281/zenodo.1256189)

There is a (separate) DOI we can use that "represents all versions and will always resolve to the latest one.: https://doi.org/10.5281/zenodo.594854

Even though zenodo has badges and citations for separate versioned DOI, I think we should stick to the one DOI that represents all versions for each of the data sets we have.

Here's info on zenodo DOI: http://help.zenodo.org/#versioning
